integration_tests/test_suites/backcompat-test-suite/test_backcompat.py

```diff
@@ -138,3 +199,45 @@ def assert_runs_and_exists(client, name):
     )
     assert len(locations) == 1
     assert locations[0].pipeline_name == name
+
+
+def test_sensor_run(graphql_client):
+    # pylint: disable=protected-access
+    repo_selector = {
+        "repositoryLocationName": "test_repo",
+        "repositoryName": "basic_repo",
+    }
+    sensors = graphql_client._execute(GET_SENSORS_QUERY, {"repositorySelector": repo_selector})
+
+    assert sensors["sensorsOrError"]["__typename"] == "Sensors"
+
+    sensor_selector = merge_dicts(
+        repo_selector,
+        {
+            "sensorName": "the_sensor",
+        },
+    )
+    response = graphql_client._execute(START_SENSOR_MUTATION, {"sensorSelector": sensor_selector})
+    assert response["startSensor"]["__typename"] == "Sensor"
+    time.sleep(5)
+
+    graphql_client._execute(
+        STOP_SENSORS_QUERY, {"jobOriginId": response["startSensor"]["jobOriginId"]}
+    )
+
+    runs_list = []
+    while are_all_runs_complete(runs_list):
+        response = graphql_client._execute(RUNS_QUERY)
+        runs_list = response["pipelineRunsOrError"]["results"]
+
+    for run in runs_list:
+        assert run["status"] == "SUCCESS"
```

Comment:
as discussed briefly offline this seems not right - we should probably wait until there is at least one run between START and STOP (as opposed to waiting 5 seconds), then here wait for them to all complete?